integration_tests/build_meta.sh

```diff
@@ -1,19 +1,14 @@
 #!/usr/bin/env bash
 
-set -e
-
-SRC=/tmp/ceresmeta-src
-TARGET=$(pwd)/ceresmeta
-
-if [[ -d ${SRC} ]] && [[ ${UPDATE_REPOS_TO_LATEST} == 'true' ]]; then
-  echo "Remove old meta..."
-  rm -rf ${SRC}
-fi
-
-if [[ ! -d ${SRC} ]]; then
-  echo "Pull meta repo..."
-  git clone --depth 1 https://github.com/ceresdb/ceresmeta.git ${SRC}
+set -o errexit
+set -o nounset
+set -o pipefail
+
+BIN_PATH=${BIN_PATH:-""}
+if [[ -z "${BIN_PATH}" ]]; then
+  echo "Fetch and install ceresmeta-server..."
+  go install github.com/CeresDB/ceresmeta/cmd/ceresmeta-server@5861780
```

Comment:
We need a formal new tag for ceresmeta to switch to `latest`.

Reply:
What's your proposal for this ref?

I'd retain `@latest` while it will be resolved to `v1.2.7` where https://github.com/CeresDB/ceresmeta/pull/265 is not included. So I wrote "We need a formal new tag".

For local debugging, the env var `BIN_PATH` can be used to specify the local version of ceresmeta-server.

Reply:
We hope that the `ceresmeta-server` used by the integration test is always the latest code, rather than based on a certain tag. Can go install do this?